metta/metta_io.py

```diff
@@ -128,24 +127,15 @@ def archive_matrix(
     if not os.path.exists(abs_path_dir):
         os.makedirs(abs_path_dir)
 
-    uuid_fname = directory + '/' + '.matrix_uuids'
-    set_uuids = load_uuids(uuid_fname)
-
     check_config_types(matrix_config)
 
     matrix_uuid = generate_uuid(matrix_config)
 
     matrix_config = copy.deepcopy(matrix_config)
     matrix_config['metta-uuid'] = matrix_uuid
 
-    write_matrix = (overwrite) or (not(matrix_uuid in set_uuids))
```

Comment:
Is there a specific reason you want to eliminate the overwrite functionality? The point of keeping track of the files in .matrix_uuids was so that this file could be pulled down from an S3 bucket without pulling the whole contents of the bucket down. 

If no one is using the part a feature I'm happy to get rid of it. I would like to keep the overwrite functionality.